libres/lib/analysis/std_enkf.cpp

```diff
@@ -218,18 +264,43 @@ bool std_enkf_set_bool(void *arg, const char *var_name, bool value) {
         bool name_recognized = true;
 
         if (strcmp(var_name, USE_EE_KEY_) == 0)
-            module_data->use_EE = value;
+            module_data->__use_EE = value;
         else if (strcmp(var_name, USE_GE_KEY_) == 0)
-            module_data->use_GE = value;
+            module_data->__use_GE = value;
         else if (strcmp(var_name, ANALYSIS_SCALE_DATA_KEY_) == 0)
             module_data->analysis_scale_data = value;
         else
             name_recognized = false;
 
+        update_inversion_enum(module_data);
         return name_recognized;
     }
 }
 
+bool std_enkf_set_string(void *arg, const char *var_name, const char *value) {
```

Comment:
Could we use `std::string` for `var_name` and `value` instead of `char *`?

Reply:
Ah, I think we can't because it is used as:

`.set_string = std_enkf_set_string,`

Is that right?

Reply:
I have not looked into it - but I think that would be a quite large change. The point is that the `std_enkf_set_string` function is called through a function pointer embedded in: [analysis_module](https://github.com/equinor/ert/blob/main/libres/lib/analysis/analysis_module.cpp) - so those changes would ripple a bit here and there. But certainly I agree with the idea, and I am happy to investigate it in a separate PR